pretrain/scripts/v3-172b-exp2-sakura/mv_unnecessaary_ckpt.py

```diff
@@ -0,0 +1,58 @@
+"""
+保存する必要のないckptディレクトリを移動するスクリプト。
+デフォルトだと1000iter刻み以外を移動。(例外として1000iter未満は残す。)
+"""
+
+import os
+from pathlib import Path
+
+import argparse
+
+def load_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--src_dir", type=str, required=True)
+    parser.add_argument("--dst_dir", type=str, required=True)
+    parser.add_argument("--min_iter", type=int, default=1000)
+    parser.add_argument("--ckpt_iter", type=int, default=1000)
+    parser.add_argument("--dryrun", action="store_true")
+    args = parser.parse_args()
+    return args
+
+def main():
+    args = load_args()
+
+    if not args.dryrun:
+        os.makedirs(args.dst_dir, exist_ok=True)
+
+    # 各iterのckptディレクトリを取得
+    src_dir = Path(args.src_dir)
+    src_ckpt_dirs = sorted(src_dir.glob("iter_*/"))
+
+    # 最後尾のckptは残す
+    src_ckpt_dirs = src_ckpt_dirs[:-1]
+    
+    for src_ckpt_dir in src_ckpt_dirs:
+        iter_num = int(src_ckpt_dir.name.split("_")[1])
+
+        # min_iter(1000iter)未満は移動の対象外とする
```

Comment:
引数で操作可能なので具体的な値を書くのは混乱の元
```suggestion
        # min_iter未満は移動の対象外とする
```